weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/HumanFeedback/ToggleColumnVisibility.tsx

```diff
@@ -0,0 +1,91 @@
+import {Switch} from '@wandb/weave/components';
+import {Button} from '@wandb/weave/components/Button';
+import {DraggableHandle} from '@wandb/weave/components/DraggablePopups';
+import {TextField} from '@wandb/weave/components/Form/TextField';
+import React, {useCallback, useState} from 'react';
+
+import {tsHumanAnnotationSpec} from './humanFeedbackTypes';
+
+type ToggleColumnVisibilityProps = {
+  columns: tsHumanAnnotationSpec[];
+  columnVisibilityModel: Record<string, boolean>;
+  setColumnVisibilityModel: (model: Record<string, boolean>) => void;
+  onEdit: (column: tsHumanAnnotationSpec) => void;
+};
+
+export const ToggleColumnVisibility: React.FC<ToggleColumnVisibilityProps> = ({
```

Comment:
is this state saved anywhere? I feel like unless it is saved / in the URL, it is sort of useless? Perhaps it sticks around while paging?